app/helpers/history_query_helper.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+# Helper methods for filtering through histories
+module HistoryQueryHelper
+  def histories_by_query(patients_identifiers)
+    History.where(patient_id: patients_identifiers.keys).order(:patient_id)
```

Comment:
when I originally designed this method, I was planning to add logic here to filter histories (and other data types) by whatever filters the user wanted to apply. Since we're not doing that though, I think it makes sense to rename this to histories_by_patient_ids

Reply:
https://github.com/SaraAlert/SaraAlert/pull/562/commits/1f13143f8968feff3c7e9ada6139b47e5cd4ecff#diff-f4e73b8e7e3c7c9d6523bd12106220b0d792d01f568f654ae5a567e7797076f9R5